requirements.txt

```diff
@@ -1,6 +1,6 @@
 django>=2.2,<3.2
 swapper>=1.1.0,<1.2.0
-djangorestframework<3.12,>=3.11
+djangorestframework~=3.12
 openwisp-utils[rest]~=0.6.3
```

Comment:
We can simply update this instead of adding `drf` and `drf-yasg` specificallly.

Reply:
@ManishShah120 When using ``openwisp-utils[rest] @ https://github.com/ManishShah120/openwisp-utils/tarball/issues/164-update-drf`` its giving error on installing dependencies. This is because openwisp-users is using ``openwisp-utils[rest]~=0.7.1``. I think we need open a PR in ``users``. @nemesisdesign am I right?

Reply:
@niteshsinha17 , I tested this branch, by pointing `openwisp-utils[rest]` to my branch, Firstly, It has to be rebased with master, and I found only one issue related with `TokenProxy`, which after fixing, all the test cases are passing.

Reply:
@niteshsinha17 yes I think you are right regarding openwisp-users, so I updated it as you suggested. It makes sense to ensure the dev versions of each module points to the dev version of the dependencies so we should start to do this regularly to avoid build and installation issues.

Reply:
> @niteshsinha17 , I tested this branch, by pointing `openwisp-utils[rest]` to my branch, Firstly, It has to be rebased with master, and I found only one issue related with `TokenProxy`, which after fixing, all the test cases are passing.

I know about this. A fresh installation was creating problems.